Makefile

```diff
@@ -71,7 +71,7 @@ build-e2e: outdir
 	[ -x _out/rte-e2e.test ] || go test -v -c -o _out/rte-e2e.test ./test/e2e/
 
 .PHONY: test-e2e
-test-e2e: build-e2e
+test-e2e: binaries build-e2e
```

Comment:
which test was broken and why we didn't see in the CI runs?

Reply:
This one: https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/master/test/e2e/rte_local/rte_local.go#L41 

We didn't see it because, since the lane PR merged on Monday and until yesterday, it ran with a static image: https://github.com/openshift-kni/resource-topology-exporter/blob/master/hack/deploy.sh#L7, instead of testing the new image which includes the changes in every PR.

Reply:
I'm not sure we want to run `rte_local` on OCP CI in the current form. This lane assumes the binary is available on the same environment on which the e2e test run, which is hard to do right on OCP CI.

Reply:
This is why I'm building the binary before running the tests and setting the correct binary path here:https://github.com/openshift-kni/resource-topology-exporter/blob/master/test/e2e/e2e_test.go#L77

Reply:
No. Everything is set. As you can see the CI with all the changes passed

Reply:
Uhm, I didn't expect it to be so easy, but I'm glad I'm wrong.

Reply:
it wasn't so easy. I did some work in the following days.